cli/Valet/Nginx.php

```diff
@@ -131,8 +148,10 @@ function ($exitCode, $outputMessage) {
     function rewriteSecureNginxFiles()
     {
         $tld = $this->configuration->read()['tld'];
+        $loopback = $this->configuration->read()['loopback'];
 
         $this->site->resecureForNewTld($tld, $tld);
+        $this->site->resecureForNewLoopback($loopback, $loopback);
```

Comment:
This is going to cause certificates to be created and deleted and then created and deleted again.

Reply:
Well, I'll refactor to keep only one method, maybe something like `resecureForNewConfiguration`.